backend/experiment/actions/base_action.py

```diff
@@ -1,9 +1,16 @@
+from .frontend_style import FrontendStyle
+
 class BaseAction(object):
     ID = 'BASE'
+    style = FrontendStyle()
 
-    def __init__(self):
+    def __init__(self, style = FrontendStyle()):
```

Comment:
If we pass the style here as an argument, there's no need to also set it as a class variable in line 5.